src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -360,6 +361,8 @@
     FLUSH_LOCAL_SCHEMA_CHANGES("cassandra.test.flush_local_schema_changes", "true"),
 
     TOMBSTONE_HISTOGRAM_TTL_ROUND_SECONDS("cassandra.streaminghistogram.roundseconds", "60"),
+
+    USE_OA_SSTABLE_FORMAT("cassandra.use_oa_sstable_format", BigFormat.TTL_MODE.LEGACY_TTL_2038.toString())
```

Comment:
I'd add some brief documentation here, including the purpose of each value and how this is  meant to be used.

Reply:
Done